### ISSUE TITLE
practice what you preach

### ISSUE THREAD

**Issue by Xiphe** [lassediercks/lassediercks-website#10]
see: https://github.com/lassediercks/lassediercks-website/blob/926031c2c56690e6dac0d871aa0b55a765e0148e/index.html#L252

VS.

https://github.com/lassediercks/lassediercks-website/commits/master

---

I really like the fun-commit style you use here, but since you use this page as an example of how you work, you might at least want to annotate that :)

**Comment by lassediercks** [comment]
Funny I just talked with @HerrBertling about this https://twitter.com/lassediercks/status/780777825511047168

I actually started with this early on https://github.com/lassediercks/lassediercks-website/commits/master?after=kmAxwsVmkObawNhxqgtVp2XgFI4rMTA0 

As I also preach to not overuse tools I decided to not stick to commit messaging format as I work alone on this.

E.g. would it be something where I absolutely want people to contribute and have a decent commit log later on I'd use the commit style format even If I work on it alone for a while.
